python/pacemaker/Makefile.am

```diff
@@ -9,4 +9,5 @@
 
 MAINTAINERCLEANFILES    = Makefile.in
 
-pkgpython_PYTHON = __init__.py
+pkgpython_PYTHON = __init__.py \
+				   exitstatus.py
```

Comment:
I like keeping buildoptions separate because it's generated from autotools.  Beyond that, I don't really have any particular reason for doing anything.

Reply:
I don't see an accepted best practice in the wild, so I guess it's whatever we want. Most of the standard library modules keep everything in the top module, but there are some major exceptions like os.

Reply:
Since this is the only public part of the API, I want to make sure we can live with our choices for a long time :) The current usage would be like

    from pacemaker.exitstatus import ExitStatus
    ... ExitStatus.OK ...

or

    import pacemaker.exitstatus
    ... pacemaker.exitstatus.ExitStatus.OK ...

That second usage is a little clunky. Looking at some installed modules, it's pretty common for `__init__.py` to import symbols from other files, so we can still keep the `.py` files separate, but have ExitStatus and BuildOptions available at the pacemaker level.